website/docs/index.html.markdown

```diff
@@ -293,6 +321,13 @@ The following arguments are used to configure the VMware Cloud Director Provider
    environment variable. This token requires at least VCD 10.3.1. There are restrictions to its use, as defined in
    [the documentation](https://docs.vmware.com/en/VMware-Cloud-Director/10.3/VMware-Cloud-Director-Service-Provider-Admin-Portal-Guide/GUID-A1B3B2FA-7B2C-4EE1-9D1B-188BE703EEDE.html)
 
+* `service_account_token_file` - (Optional; *v3.9+*) This is the file that contains a Service Account API token. The
+   path to the file could be provided as absolute or relative to the working directory. It is used instead of username
+   and password (in combination with `auth_type=service_account_token_file`. The file can also be specified with the 
+   `VCD_SA_TOKEN_FILE` environment variable. This option requires at least VCD 10.4.0. 
```

Comment:
Let's move required VCD version to the parenthesis, like here:
https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/vapp_vm#sizing_policy_id